### ISSUE TITLE
(EDDI docked)-event not working in V. 3.0.1-b3

### ISSUE THREAD

**Issue by nepomuk16321** [EDCD/EDDI#578]
## EDDI Version 3.0.1-b3 as VA-plugin 

I have tested version 3.0.1-b3 for translation (german). First uninstalled version 3.0.1-b1 then installed version 3.0.1-b3 . Test in ELITE (EDDI docked)-event not working. Then uninstalled V3.0.1-b3 and installed 3.0.1-b1 and the event works again. In V3.0.1-b3 the variables {TXT:EDDI docked system} and {TXT:EDDI docked station} are NOT SET.

Thx for your help, nepo

**Comment by hoksilato2** [comment]
same here :)

**Comment by Tkael** [comment]
Ok. So I understand that some variables are not being set in VoiceAttack. Does the event work with EDDI in standalone (without running VoiceAttack)? Or is the event simply not being triggered at all?

**Comment by Hoodathunk** [comment]
Yup, ((EDDI docked)) not triggering in VA. In particular, not showing up in event queue.

**Comment by Tkael** [comment]
```
2018-05-22T05:54:39 ScriptResolver:resolveScript [W] Failed to resolve the: Docked script. Cottle.Exceptions.ParseException: expected 'end of block', found ' '
   at Cottle.Parsers.DefaultParser.ParseCommand()
   at Cottle.Parsers.DefaultParser.Parse(TextReader reader)
   at Cottle.Parsers.PostParser.Parse(TextReader reader)
   at Cottle.Documents.SimpleDocument..ctor(TextReader reader, ISetting setting)
   at Cottle.Documents.SimpleDocument..ctor(String template, ISetting setting)
   at EddiSpeechResponder.ScriptResolver.resolveScript(String script, BuiltinStore store, Boolean master, Script scriptObject) in EDDI\SpeechResponder\ScriptResolver.cs:line 85 
```

**Comment by hoksilato2** [comment]
The event ((EDDI docked)) simply not being triggered at all

**Comment by Tkael** [comment]
```
2018-05-22T06:38:02 EDDI:OnEvent [W] Responder failed System.InvalidOperationException: Nullable object must have a value.
   at System.ThrowHelper.ThrowInvalidOperationException(ExceptionResource resource)
   at System.Nullable`1.get_Value()
   at EDDNResponder.EDDNResponder.sendCommodityInformation() in EDDI\EDDNResponder\EDDNResponder.cs:line 229
   at EDDNResponder.EDDNResponder.handleDockedEvent(DockedEvent theEvent) in EDDI\EDDNResponder\EDDNResponder.cs:line 183
   at EDDNResponder.EDDNResponder.Handle(Event theEvent) in EDDI\EDDNResponder\EDDNResponder.cs:line 85
   at Eddi.EDDI.<>c__DisplayClass101_1.<OnEvent>b__0() in EDDI\EDDI\EDDI.cs:line 721
```

**Comment by Tkael** [comment]
Confirmed. While the EDDN responder is enabled, the event throws an exception and does not trigger. If I disable the EDDN responder and attempt to dock again then the event does trigger.

**Comment by Djuwanna** [comment]
I tried disabling the EDDN responder in EDDI and it's still not recognizing the event for me :(

**Comment by Hoodathunk** [comment]
Update: We have the cause identified and a working solution. Expect to see the fix in upcoming b4 release.

**Comment by richardbuckle** [comment]
Tested OK. Fix will be in 3.0.1-b4.
